### PR TITLE
Refactor: Rename pipeline into stream

### PR DESCRIPTION
**Pull request by ph**
To align the code base with the multiple output strategy we will rename
the pipeline concept into the streams.

See elastic/beats#14445

### CODE REVIEW

**Comment by ph** [comment]
jenkins test this please

**Comment by ph** [comment]
thats odd all theses conflicts..

**Comment by ph** [comment]
lets get that merged. I am working on making the configurations changes work.

**Comment by michalpristas** [comment]
can you recreate this PR? i find this diff really disturbing. something is off here

**Comment by ph** [comment]
You are right, this is weird. let me try to rebase it off a fresh fleet branch.

File: x-pack/agent/pkg/agent/application/stream.go [hunk 1]
```diff
@@ -40,28 +40,28 @@ type sender interface {
...
-func (b *operatorPipeline) Execute(cfg *configRequest) error {
+func (b *operatorStream) Execute(cfg *configRequest) error {
 	return b.configHandler.HandleConfig(cfg)
 }
 
-func pipelineFactory(cfg *config.Config, client sender, r reporter) func(*logger.Logger, routingKey) (pipeline, error) {
-	return func(log *logger.Logger, id routingKey) (pipeline, error) {
+func streamFactory(cfg *config.Config, client sender, r reporter) func(*logger.Logger, routingKey) (stream, error) {
+	return func(log *logger.Logger, id routingKey) (stream, error) {
 		// new operator per pipeline to isolate processes without using tags
```
Comment (michalpristas) [inline x-pack/agent/pkg/agent/application/stream.go]: s/per pipeline/per stream

**Comment by ph** [comment]
fixed waiting for CI.

**Pull request merged**